src/lib/utils/transferProps.js

```diff
@@ -0,0 +1,11 @@
+const ALWAYS_BLACKLISTED_PROPS = ['className', 'forwardedRef'];
+
+export default (props, blacklistedProps) => {
```

Comment:
For unknown reason, `(props, blacklistedProps = [])` does not work here. That's the reason I wrote `(blacklistedProps || [])` bellow. (No time stuck on this)